web/src/pages/Main.tsx

```diff
@@ -8,6 +10,26 @@ import { ComponentResponse, MultiflashResponse } from '../api/generated'
 import { ComponentSelector } from '../components/feature/ComponentSelector'
 import { PhaseTable } from '../components/feature/PhaseTable'
 
+const Results = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  gap: 50px;
+  align-items: center;
+  flex-wrap: wrap;
```

Comment:
This behaves a bit weird when resizing the window (i.e. dev tools). It wraps when the window becomes to small to have both tables on the same row as expected, but when expanding it puts them on the same row until some point where it wraps, even though there is enough space, before putting them on the same row again.
I think it has something to do with min-width: 1200px in Container.